auth_oauth_block_password_login/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Quartile Limited (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    allow_oauth_domains = fields.Text()
```

Comment:
`force_oauth_domains` is probably a more suitable name.  Please explain the domains should be separated in the field (by comma, by line break or something else).